File: servico-musica-java/java.grpc/src/main/java/servico/java/grpc/repository/PlaylistRepository.java
package servico.java.grpc.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import servico.java.grpc.entity.Playlist;

import java.util.List;

@Repository
public interface PlaylistRepository extends JpaRepository<Playlist,Long> {
    List<Playlist> findByMusicasId(Long idMusica);
}

File: servico-musica-kotlin/kotlin.grpc/src/main/kotlin/servico/kotlin/grpc/repository/PlaylistRepository.kt
package servico.kotlin.grpc.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import servico.kotlin.grpc.entity.Playlist

@Repository
interface PlaylistRepository : JpaRepository<Playlist, Long> {
    fun findByMusicasId(idMusica: Long?): MutableList<Playlist>
}
